packages/v4l_utils.rb

```diff
@@ -1,42 +1,25 @@
-require 'package'
+require 'buildsystems/meson'
 
-class V4l_utils < Package
+class V4l_utils < Meson
   description 'The v4l-utils are a series of packages for handling media devices.'
   homepage 'https://www.linuxtv.org/wiki/index.php/V4l-utils'
-  version '1.22.1'
+  version '1.28.1'
   license 'GPL-2+ and LGPL-2.1+'
-  compatibility 'x86_64 aarch64 armv7l'
-  source_url "https://linuxtv.org/downloads/v4l-utils/v4l-utils-#{version}.tar.bz2"
-  source_sha256 '65c6fbe830a44ca105c443b027182c1b2c9053a91d1e72ad849dfab388b94e31'
+  compatibility 'all'
+  source_url 'https://git.linuxtv.org/v4l-utils.git'
+  git_hashtag "v4l-utils-#{version}"
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '99bfc727cc7dc8669f57e981f8659e8278007e61c9425101f5ea6d27e3d9c2b1',
-     armv7l: '99bfc727cc7dc8669f57e981f8659e8278007e61c9425101f5ea6d27e3d9c2b1',
-     x86_64: '10c0384638fef3657f01129ce225a640ac182823f472d35aa48bfc3a39da7a5f'
+    aarch64: '6f9649d3b4ab8bad29abc672da5aaecd451a011a1b53f302b0f1edb0350ae387',
+     armv7l: '6f9649d3b4ab8bad29abc672da5aaecd451a011a1b53f302b0f1edb0350ae387',
+       i686: 'b0ff283ea4749e07c74d5a7923b1a1247795546f9af78ba1e0c91962ad32c386',
+     x86_64: '9abf83c04426cae26ef5eccdd1c238fdb6d1c8f6e8d49e965a76ac0f336d1314'
   })
 
-  depends_on 'alsa_lib'
-  depends_on 'eudev'
-  depends_on 'libglu'
   depends_on 'libjpeg_turbo'
-  depends_on 'mesa' unless ARCH == 'i686'
-  depends_on 'qt5_base' unless ARCH == 'i686'
-  depends_on 'sdl2_image'
-  depends_on 'gcc_lib' # R
-  depends_on 'glibc' # R
```

Comment:
Doesn't depend upon glibc?

Reply:
Are we certain all these other dependencies are not needed?

Reply:
yup